test/e2e/specs/specs-playwright/wp-plans_purchases-spec.ts

```diff
@@ -0,0 +1,119 @@
+import {
+	DataHelper,
+	LoginFlow,
+	setupHooks,
+	SidebarComponent,
+	PlansPage,
+	IndividualPurchasePage,
+	CartCheckoutPage,
+	BrowserHelper,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+/**
+ * Constants
+ */
+
+const userOnPremiumPlan = 'defaultUser';
+
+describe( DataHelper.createSuiteTitle( 'Plans: Purchases' ), function () {
+	let page: Page;
+	let plansPage: PlansPage;
+	let purchasesPage: IndividualPurchasePage;
+	let cartCheckoutPage: CartCheckoutPage;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Initial navigation', function () {
+		it( 'Log In', async function () {
+			const loginFlow = new LoginFlow( page, userOnPremiumPlan );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Navigate to Plans from sidebar', async function () {
+			const sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.gotoMenu( { item: 'Upgrades', subitem: 'Plans' } );
+		} );
+
+		it( 'Click on the "My Plan" tab', async function () {
+			plansPage = new PlansPage( page );
+			await plansPage.clickNavigationTab( 'My Plan' );
+		} );
+
+		it( 'The Premium plan is listed as current plan on "My Plan" tab', async function () {
+			await plansPage.validatePlanInMyPlan( 'Premium' );
```

Comment:
Just found the name funny 😆 

`plan in plan`

Reply:
Ha, right? I couldn't come up with anything better 😆  So I'm very open to ideas!

Reply:
Since the code is validating that the current plan that is active in the `My Plans` tab, if you want to change it my suggestions would be :

- validateActivePlan
- checkActivePlan

and a couple other variations on the word `check/validate`.

Correct me if I'm wrong, but it seems that `active plan` is only shown within Upgrades >> Plans : Plans tab, so I think it's unnecessary to specify `in my plan`.

Reply:
>Correct me if I'm wrong, but it seems that active plan is only shown within Upgrades >> Plans : Plans tab, so I think it's unnecessary to specify in my plan.

Well, there is really two ways to validate this: the titular card in the "my plan" tab, and whichever card has "manage" in the "plans" tab, so I wanted some distinction.

It's kind of verbose, but what about "validateActivePlanInMyPlanTab"?